romana-install/group_vars/aws

```diff
@@ -1,6 +1,12 @@
-key_name: "shared-key-1"
-
+ec2_keypair: ""
 region: "us-west-1"
-instance_type: "t2.large"
+instance_type:
+  controller:
+    devstack: "t2.large"
+    kubernetes: "t2.small"
+  compute:
+    devstack: "t2.small"
+    kubernetes: "t2.small"
+
 controller_image: "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
```

Comment:
We can't have the region as a configurable parameter (or mention it as an option) without also letting people know that the AMIs need to be changed as well (since they only exist in one region). What I have usually seen is that you create a map, with the correct AMIs for each region, and then use that map to get the AMI in the playbooks, based on the chosen region.